### DISCUSSION TITLE
Battery Health Graph Not Updating?

### DISCUSSION THREAD

**Discussion by jcabaleiro** [teslamate-org/teslamate#4015] [Q&A]
We have two Teslas on Teslamate.  The battery health graph does not seem to be updating for one of the cars, in spite of multiple charges both at home and on the road.  Any ideas why this might be?

Also, one of the cars is LFP battery, I just updated and LFP battery health panel no longer there, has battery health been consolidated to one panel?

<img width="1688" alt="image" src="https://github.com/teslamate-org/teslamate/assets/24703328/45f2b94c-d327-4b4f-984d-5b301935a732">

**Comment by jcabaleiro** [comment]
Follow up info.  I found that on June 15, where the graph ends, I had a bunch of incomplete charges. I followed instructions here to terminate the incomplete charges (my mobile charger was having some issues).   This did not fix problem, so I went ahead and deleted all June 15 charges (there where 3 completed ones due to charger issues).  Still no fix.

**Comment by 07stuntar1** [comment]
I think im having some what of the same issue. I have 2 cars but both are not recording data. It worked perfectly fine on an older update. https://github.com/teslamate-org/teslamate/pull/4007#issuecomment-2198602812 is saying I could be using different databases but no one says how to fix it.
